pkg/syncer/spec/spec_controller.go

```diff
@@ -43,6 +44,7 @@ import (
 
 const (
 	controllerName                   = "kcp-workload-syncer-spec"
+	byNamespaceLocatorIndexName      = "syncer-spec-ByNamespaceLocator-index"
```

Comment:
nit: all of these are indexes. No need for the `-index` postfix.

Reply:
done